radicle-cli/src/commands/auth.rs

```diff
@@ -94,11 +104,13 @@ pub fn init(options: Options) -> anyhow::Result<()> {
     let profile = Profile::init(home, passphrase.clone())?;
     spinner.finish();
 
-    let spinner = term::spinner("Adding your radicle key to ssh-agent...");
-    if register(&profile, passphrase).is_ok() {
-        spinner.finish();
-    } else {
-        spinner.warn();
+    if let Some(mut agent) = connect_ssh_agent()? {
+        let spinner = term::spinner("Adding your radicle key to ssh-agent...");
+        if register(&mut agent, &profile, passphrase).is_ok() {
+            spinner.finish();
+        } else {
+            spinner.warn();
+        }
     }
```

Comment:
If can't connect, we should warn the user that we couldn't connect.

`warn("Could not connec to ssh-agent")`